user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectDetailsComponent.java

```diff
@@ -226,8 +226,6 @@ private void openProjectInformationDialog() {
     project.ifPresentOrElse(proj -> {
           EditProjectInformationDialog editProjectInformationDialog = generateEditProjectInformationDialog(
               proj);
-          editProjectInformationDialog.addCancelListener(
```

Comment:
why could this be removed without replacement?

Reply:
Because the closing is handled by the dialog already. No special method is needed anymore as the close method resets the dialog.